tests/simulationbase.py

```diff
@@ -0,0 +1,169 @@
+from pathlib import Path
+import abc
+import json
+import random
+import shutil
+
+
+class SimulationBase:
```

Comment:
Shouldn't it inherit from `ABC`?

Reply:
What does it change?

Reply:
[The docs](https://docs.python.org/3/library/abc.html#abc.abstractmethod) state that you should use it:
>  @abc.abstractmethod
> ...
> Using this decorator requires that the class’s metaclass is ABCMeta or is derived from it.
But I never really investigated it.